it-operations/it-operations-quiz.md

```diff
@@ -568,3 +568,17 @@ To get to either UEFI or BIOS on a system you usually go via the computer's boot
 - [ ] It has been integrated into the southbridge to provide support for the CPU.
 - [ ] It is the peripheral and I/O controller, connected to the processor via the southbridge.
 - [ ] It provides mathematical and 3D physics support for the CPU in the absence of a GPU.
+
+#### Q69. Which characters are not allowed in a filename?
```

Comment:
Maybe it is worth mentioning the exact OS you speak about.

Reply:
> Maybe it is worth mentioning the exact OS you speak about.

While I would usually agree with that, I think the options provided below the question already do a decent job of giving the user an idea of which OS is being referenced here, which is why I didn't see a need to further specify the OS. Thank you for pointing it out, though!

Reply:
Yeah, maybe you right